husky_gazebo/launch/husky_ur5.launch

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```

Comment:
Chain with the existing description_gazebo.launch file. This way the arm stuff can be first-class in simulation and integrated with the rest of husky bringup. Use the ur5_enabled arg to pass down into xacro